src/app/media/[id]/page.tsx

```diff
@@ -0,0 +1,34 @@
+import { getMediaDetails } from "@/api/media";
+import Image from "next/image";
+import PlayIcon from "#/icons/home/play.svg";
+
+interface Params {
+  id: number;
+}
+
+interface DetailPageProps {
+  params: Params;
+}
+
+const DetailPage = async ({ params }: DetailPageProps) => {
+  const posterBaseUrl = process.env.NEXT_PUBLIC_POSTER_BASE_URL;
+  const data = await getMediaDetails(params.id);
+  return (
+    <div className="flex-center flex-column gap-32pxr">
+      <div className="relative h-415pxr w-full">
+        <Image src={`${posterBaseUrl}${data.poster_path}`} alt={`영화 포스터 이미지: ${data.title}`} fill />
+        <div className="absolute inset-0pxr bg-gradient-to-b"></div>
```

Comment:
요 부분도 컴포넌트로 따로 생성해주면 코드가 좀 더 깔끔해질 것 같다는 개인적인 의견입니다 ㅎㅎ

```suggestion
  <ImageGradient />
```